src/CMakeLists.txt

```diff
@@ -4,21 +4,29 @@ else()
     add_definitions(-DIS_DEBUG=0)
 endif()
 
-option(DISABLE_JIT_SUPPORT "Build without JIT support" OFF)
+# Provide ENABLE_JIT_SUPPORT option and automatically disable JIT if compiler is not Clang
+option(ENABLE_JIT_SUPPORT "Build with JIT support" ON)
 if (NOT ${LLVM_FOUND} OR NOT "${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang" OR CMAKE_CXX_COMPILER_VERSION VERSION_LESS 5.0)
-    set(DISABLE_JIT_SUPPORT ON)
+    set(ENABLE_JIT_SUPPORT OFF)
 endif()
 
-if (NOT DISABLE_JIT_SUPPORT)
```

Comment:
Though cmake allows passing variable names to if, i feel safer just expanding the value here using `${...}`